examples/arithmetics/esbuild.mjs

```diff
@@ -0,0 +1,26 @@
+//@ts-check
+import * as esbuild from 'esbuild';
+
+const watch = process.argv.includes('--watch');
+
+const ctx = await esbuild.context({
+    entryPoints: ['src/extension.ts', 'src/language-server/main.ts'],
+    outdir: 'out',
+    outExtension: {
+        '.js': '.cjs'
+    },
+    bundle: true,
+    target: "es6",
+    format: 'cjs',
+    loader: { '.ts': 'ts' },
+    external: ['vscode'],
+    platform: 'node',
+    sourcemap: true
+});
+
+if (watch) {
+    await ctx.watch();
+} else {
+    await ctx.rebuild();
```

Comment:
Errors are printed to the console in case they are encountered. If we were to print every successful build, we would have a lot of cluttering in the console, since we start 5 esbuild processes when running `npm run watch`.

Reply:
> we would have a lot of cluttering in the console

Yes, I know. It is about consistency with tsc behaviour. But also not worth discussing further. 🙂 Up to you.